controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -454,7 +460,12 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
     },
 
     getFederatedGraphByName: (req, ctx) => {
-      return getFederatedGraphByName(opts, req, ctx);
+      const request: GetFederatedGraphRequest = new GetFederatedGraphRequest({
+        name: req.name,
+        includeMetrics: req.includeMetrics,
+        namespace: req.namespace,
+      });
+      return getFederatedGraph(opts, request, ctx);
```

Comment:
We should try to build specialized interfaces like getFederatedGraphByName instead of supporting optional arguments.